Design-Docs/Image-Properties-Spec.md

```diff
@@ -211,3 +211,25 @@ A boolean property that is not present is considered to be `false`.
   pricing (but it might be provided by a contracted 3rd party, e.g. the OS vendor).
 
 
+##  Standardized Image  
+
+Images which are "default" in Glance of any SCS-Provider and are certified by SCS, should have following Features:
+
+*   Optional: Latest LTS Distribution of:
+ -    Debian
+ -    Ubuntu, 
+ -    AlmaLinux and/or Rocky Linux
+ -    OpenSLES
+
+* Optional: A generic Kernel which is running on every Environment
+
+* Needed: Hardening: an enabled Mandatory Access Control System, either SELinux or Apparmor
+*   Optional: a definite User specification for the default Login User, not Distribution specified.
+* Needed: An only ssh-key enabled Authentication.
+* Needed: a Pipeline for building images with verification key 
+* Needed: a Pipeline pass through (install something, an test function)
+* Needed: a Mirror to hosted SCS-Images
+* Optional: default software tools
+    - tbd
+
```

Comment:
The thoughts on building optimized images should -- IMVHO -- not be part of a specs file at this point.
Note that the Image-Properties-Spec.md file IS an official SCS standard -- thoughts on how to eveolve it should be tracked in a separate document.
While I think it's appropriate to suggest (or even mandate?) that the provider maintains a set of the most popular free Linux images as part of the spec, we should not discuss the early-stage ideas here in the same doc, IMVHO.